apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -34,7 +34,7 @@
 % Swagger
 
 -define(API_TAGS_GLOBAL, [<<"Authentication">>]).
--define(API_TAGS_SINGLE, [<<"Listener authentication">>]).
```

Comment:
I'm curious why do this, in our docs, the format of the title is `Foo Boo`, not `Foo boo`